### PR TITLE
CTRL- and SHIFT- click features in Calypso

### PR DESCRIPTION
**Pull request by majella67**

### CODE REVIEW

File: General/InterestingsToKnowForBeginners.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ The goal is to ease the use of Pharo for beginners.
 - [API](#api)
 - [Evaluatable objects (Blocks and Symbols)](#evaluatable-objects--blocks-and-symbols-)
 - [Tools](#tools)
+  * [Calypso System Browser](#calypso)
   * [Dependency Analyzer](#dependency-analyzer)
+
```
Comment (juliendelplanque) [inline General/InterestingsToKnowForBeginners.md]: Is this extra newline really needed?

File: General/InterestingsToKnowForBeginners.md [hunk 2]
```diff
@@ -67,6 +69,20 @@ The second way is shorter and often more readable.
 
 > TODO
 
+### Calypso (System browser)
+
+- Display the differences between two methods
+	* Click on a method
+	* Hold down the SHIFT-key
```
Comment (juliendelplanque) [inline General/InterestingsToKnowForBeginners.md]:
```suggestion
	* Hold down the `SHIFT` key
```

File: General/InterestingsToKnowForBeginners.md [hunk 3]
```diff
@@ -67,6 +69,20 @@ The second way is shorter and often more readable.
...
+	* Click on the second method
+	* Click on the tab Diff to see the differences
+	
+- Editing 2 methods side by side
+	* Click on a method
+	* Hold down the CTRL-key
```
Comment (juliendelplanque) [inline General/InterestingsToKnowForBeginners.md]:
```suggestion
	* Hold down the `CTRL` key
```

File: General/InterestingsToKnowForBeginners.md [hunk 4]
```diff
@@ -67,6 +69,20 @@ The second way is shorter and often more readable.
...
 
+### Calypso (System browser)
+
+- Display the differences between two methods
+	* Click on a method
+	* Hold down the SHIFT-key
+	* Click on the second method
+	* Click on the tab Diff to see the differences
+	
+- Editing 2 methods side by side
```
Comment (juliendelplanque) [inline General/InterestingsToKnowForBeginners.md]: It works with as many methods as you want by the way :-)

Maybe it is worth to specify it.

Comment (majella67) [reply General/InterestingsToKnowForBeginners.md]: Done

**Pull request merged**